airflow/providers/amazon/aws/transfers/mysql_to_s3.py

```diff
@@ -82,6 +86,8 @@ def __init__(
         pd_csv_kwargs: Optional[dict] = None,
         index: bool = False,
         header: bool = False,
+        file_format: str = 'csv',
```

Comment:
A `Literal` annotation would be useful here.

Reply:
Done